### ISSUE TITLE
Should the player be able to name him or herself? 

### ISSUE THREAD

**Issue by christiancosgrove** [BoolClub/ProjectCrusade#49]
This is something many RPGs share. If we do allow for custom names, we could insert the player's name into NPC dialogues.

**Comment by Authman2** [comment]
Yeah, I think that would be a good idea. It would also make the game a bit more playable.
